### ISSUE TITLE
Composer REST Server starts with an error - indication of changed dependency?

### ISSUE THREAD

**Issue by rthatcher** [hyperledger/composer#1733]
## Context
Composer REST Server starts with an error:
**(node:4578) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: cb is not a function**

## Expected Behavior
It should start without error

## Actual Behavior
Although it shows the error the REST server does generate the API and using simple samples it appears to function as expected.

## Steps to Reproduce
Repeatable on:
- Ubuntu 16.04
- node v6.11.0
- npm v5.1.0
- composer v0.10.1

Following commands run:
```
npm uninstall -g composer-rest-server
npm install -g composer-rest-server
composer-rest-server -p hlfv1 -n my-network -i PeerAdmin -s randomString -N never
```

**Comment by davidkel** [comment]
This is likely to be a dependency has shipped a new version, which may or may not have broken some part of the rest server. This needs to be tracked down. 
This is becoming a common occurance and we really need to fix the versions we depend on and the versions all the dependencies are using.

**Comment by davidkel** [comment]
This was caused by loopback-boot version change from 2.25 to 2.26. Have fixed our rest server version to 2.25 plus any micro changes for bug fixes (ie ~2.25.0) in package.json
